src/index.html

```diff
@@ -3,12 +3,53 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#Home">
+        <img
+        src="./images/logo.png"
+        alt="Company logo"
+        class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="menuLink is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="menuLink is-active">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="menuLink is-active"
+              data-qa="hover"
+            >Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="menuLink is-active">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="menuLink is-active">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="menuLink is-active">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="menuLink is-active">Video</a>
+          </li>
```

Comment:
Add line breaks between child tags, to improve readability
```suggestion
          </li>
          <li class="nav__item">
            <a href="#" class="menuLink is-active">Gadgets</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="menuLink is-active">Tablets</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="menuLink is-active">Photo</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="menuLink is-active">Video</a>
          </li>
```